src/com/clover/remote/client/transport/websocket/WebSocketCloverTransport.ts

```diff
@@ -137,7 +129,11 @@ export abstract class WebSocketCloverTransport extends CloverTransport implement
                 // Hold the message in case we need to put it back on the queue
                 let nextMsg: string = this.messageQueue.shift();
                 try {
-                    this.webSocket.send(nextMsg);
+                    if (this.webSocket.getBufferedAmount() > 0) {
```

Comment:
Not 100% sure about this, It seems like it is the right thing to do and appears to make things more stable in my testing.

Reply:
See above question on functionality check.